projects/packages/post-list/src/class-post-list.php

```diff
@@ -37,9 +37,13 @@ public static function get_instance() {
 	public function register() {
 		if ( ! did_action( 'jetpack_on_posts_list_init' ) ) {
 			add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
-			add_action( 'current_screen', array( $this, 'add_filters_and_actions' ) );
+			add_action( 'current_screen', array( $this, 'add_filters_and_actions_for_screen' ) );
 			add_filter( 'default_hidden_columns', array( $this, 'adjust_default_columns' ), 10, 2 );
 
+			if ( wp_doing_ajax() && ! empty( $_POST['action'] ) && 'inline-save' === $_POST['action'] && wp_verify_nonce( 'inlineeditnonce', '_inline_edit' ) ) {
+				add_action( 'admin_init', array( $this, 'add_filters_and_actions_quick_edit' ) );
```

Comment:
This register method is already being executed via the `admin_init` action, so I don't _think_ this will run